src/reactions/MessageReactionList.js

```diff
@@ -0,0 +1,58 @@
+/* @flow strict-local */
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { View } from 'react-native';
+import type { NavigationScreenProp } from 'react-navigation';
+
+import type { AggregatedReaction, GlobalState, Message, User } from '../types';
+import { Screen } from '../common';
+import { getSelfUserDetail } from '../selectors';
+import aggregateReactions from './aggregateReactions';
+import styles from '../styles';
+import { getUsersById } from '../users/userSelectors';
+import getReactionsTabs from './ReactionListTabView';
+
+type Props = {
+  message: Message,
+  navigation: NavigationScreenProp<*> & {
+    state: {
+      params: {
+        src: string,
+        message: Message,
+      },
+    },
+  },
+  ownUserId: number,
+  users: Map<number, User>,
+};
+
+class MessageReactionList extends PureComponent<Props> {
+  render() {
+    const { message, navigation, ownUserId, users } = this.props;
+    const { initialRouteName } = navigation.state.params;
```

Comment:
Wait, is it `src` or `initialRouteName`?

Reply:
Type was wrong, it is `initialRouteName` 😞